src/dmifn.rs

```diff
@@ -589,15 +589,14 @@ pub fn dmi_processor_id(data: &SMBiosProcessorInformation<'_>) {
                     2 => {
                         println!(
                             "\tSignature: Family {}, Model {}, Stepping {}",
-                            (eax >> 8)
-                                & 0xF
+                            ((eax >> 8) & 0xF)
                                     + match ((eax >> 8) & 0xF) == 0xF {
-                                        true => (eax >> 20) & 0xFF,
+                                        true => ((eax >> 20) & 0xFF),
```

Comment:
The parentheses here inside the true match arm have no effect (no other operators to resolve precedence with) and their presence may be confusing.